tv_apps/browser/js/firefox_account.js

```diff
@@ -52,8 +52,11 @@
       }.bind(this), this.onStatusError.bind(this));
   };
 
-  FirefoxAccount.prototype.signOut = function firefoxAccount_signOut () {
-    // XXX: Implement sign out
+  FirefoxAccount.prototype.signOut = function firefoxAccount_signOut (cb) {
+    FxAccountsIACHelper.logout(() => {
+      this.onStatusChange();
```

Comment:
This line doesn't work because of missing an argument. And I guess this line is not needed. ```onStatusChange``` will be called once the module receives an ```onlogout``` event.